ci-operator/step-registry/gather/extra/gather-extra-commands.sh

```
@@ -138,6 +138,7 @@ queue ${ARTIFACT_DIR}/releaseinfo.json oc --insecure-skip-tls-verify --request-t
 queue ${ARTIFACT_DIR}/clusterrolebindings.json oc --insecure-skip-tls-verify --request-timeout=5s get clusterrolebindings --all-namespaces -o json
 queue ${ARTIFACT_DIR}/networkpolicies.json oc --insecure-skip-tls-verify --request-timeout=5s get networkpolicies --all-namespaces -o json
 queue ${ARTIFACT_DIR}/oc_cmds/networkpolicies oc --insecure-skip-tls-verify --request-timeout=5s get networkpolicies --all-namespaces
+queue ${ARTIFACT_DIR}/pacemakercluster-cluster.json oc --insecure-skip-tls-verify --request-timeout=5s get pacemakerclusters cluster -ojson
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)